Makefile

```diff
@@ -22,7 +22,7 @@ all: test
 ##@ Build
 .PHONY: build
 build: ## Go build
-	go build -o cip ${REPO_ROOT}/cmd/cip/...
+	${REPO_ROOT}/go_with_version.sh build ${REPO_ROOT}/cmd/cip
```

Comment:
This should have been merged with #324 
When building, we currently have an incomplete version stamping.
```bash
$ cip version
GitVersion:    
GitCommit:     
GitTreeState:  
BuildDate:     
GoVersion:     go1.16.5
Compiler:      gc
Platform:      linux/amd64
```

With this change, we get all the information we want.
```bash
$ cip version
GitVersion:    v3.0.0
GitCommit:     999f2538d2d4a63a1b5c389d7b53411e9ad24226
GitTreeState:  clean
BuildDate:     2021-07-07T21:06:08Z
GoVersion:     go1.16.5
Compiler:      gc
Platform:      linux/amd64
```